build.gradle.kts

```diff
@@ -165,7 +165,8 @@ idea {
 }
 
 application {
-	mainClassName = "edu.duke.cs.osprey.control.Main"
+	mainClassName = "edu.duke.cs.osprey.design.Main"
+	applicationDefaultJvmArgs = listOf("--add-modules=jdk.incubator.foreign")
```

Comment:
I think these args are the same as `moduleArgs` on line 141? Could remove the duplication if you want.

Reply:
Removed